traffic_prophet/countmatch/tests/test_growthfactor.py

```diff
@@ -1,108 +1,165 @@
+import pytest
 import hypothesis as hyp
 import numpy as np
-import pandas as pd
 import statsmodels.api as sm
 
-from .. import reader
+from .. import permcount as pc
+from .. import derivedvals as dv
 from .. import growthfactor as gf
 
-from ...data import SAMPLE_ZIP
 
+def get_single_ptc(sample_counts, cfgcm_test, count_id):
+    pcpp = pc.PermCountProcessor(None, None, cfg=cfgcm_test)
+    perm_years = pcpp.partition_years(sample_counts.counts[count_id])
+    ptc = pc.PermCount.from_count_object(sample_counts.counts[count_id],
+                                         perm_years)
+    dvs = dv.DerivedVals('Standard')
+    dvs.get_derived_vals(ptc)
+    return ptc
 
-class TestGrowthFactor:
-    """Test growth factor calculation."""
+
+@pytest.fixture(scope='module')
+def ptc_oneyear(sample_counts, cfgcm_test):
```

Comment:
As with `derivedvals`, only ` TestGrowthFactorBase` uses these, so move them into the class.  Don't combine them, since WADT calculations behave differently for multi-year PTCs.